### DISCUSSION TITLE
尝试SM.MS/GitHub图床上传均返回false，日志显示Can't find aliYun OSS config

### DISCUSSION THREAD

**Discussion by RainGiving** [Molunerfinn/PicGo#634] [Q&A]
关键是没有使用过阿里云OSS图床，不知道错误原因在哪，除Github图床外其他均为默认配置
![image](https://user-images.githubusercontent.com/71825807/109026127-5c74eb80-76fa-11eb-8968-d387d9561668.png)

**Comment by Molunerfinn** [comment]
查看配置文件里， `picBed.current` 以及 `picBed.uploader` 配置是什么，如果不是 `github` 或者 `smms` 可以手动修改一下，然后重启picgo看看。另外使用的是什么版本的picgo?以前某个版本会有这个问题，可以升级一下最新的

**Reply by RainGiving** [reply]
修改`picBed.uploader`为`github`可解决问题，感谢
